integration/tests/cook/test_basic.py

```diff
@@ -25,42 +25,56 @@ def setUp(self):
         util.wait_for_cook(self.cook_url)
 
     def test_basic_submit(self):
-        job_uuid, resp = util.submit_job(self.cook_url)
-        self.assertEqual(resp.status_code, 201)
+        job_executor_type = util.get_job_executor_type(self.cook_url)
+        job_uuid, resp = util.submit_job(self.cook_url, executor=job_executor_type)
```

Comment:
Why do we need to set the `executor` to the value that's already configured?

Reply:
To enable the test on line 35 to be triggered.

Reply:
Should we make `util.submit_job` do this by default so that we don't have to modify so many integration tests?

Reply:
It looks like 6 tests had to add 

```python
job_executor_type = util.get_job_executor_type(self.cook_url)
..., executor=job_executor_type)
```

Will new tests need this as well? I'm wondering if there is a way to avoid having this in several tests

Reply:
Only tests that care about using the cook executor. All of the tests that specify it, also have a check later for a few assertions based on `if job_executor_type == 'cook':`

Reply:
OK